fhir-bulkdata-webapp/src/test/java/com/ibm/fhir/jbatch/bulkdata/tools/Main.java

```diff
@@ -0,0 +1,339 @@
+/*
+ * (C) Copyright IBM Corp. 2020
```

Comment:
Is this file needed, since it is commented out?  Otherwise, update copyright.

Reply:
I'm going to remove it.  It was a prior test file.